addons/crm/models/res_partner.py

```diff
@@ -52,6 +52,11 @@ def _compute_opportunity_count(self):
                     partner.opportunity_count += group['partner_id_count']
                 partner = partner.parent_id
 
+    @api.depends('parent_id')
+    def _compute_team_id(self):
+        for partner in self.filtered(lambda rec: not rec.team_id and rec.parent_id.team_id and rec.company_type == 'person'):
+            partner.team_id = partner.parent_id.team_id
```

Comment:
You need to have a default value to False to prevent "cache miss" error
```suggestion
    def _compute_team_id(self):
        for partner in self:
            if not partner.team_id and partner.parent_id.team_id and partner.company_type == 'person':
                partner.team_id = partner.parent_id.team_id
            elif not partner.team_id:
                partner.team_id = False
```